src/Management/test/Endpoint.Test/Env/EndpointMiddlewareTest.cs

```diff
@@ -64,6 +64,8 @@ public async Task EnvActuator_ReturnsExpectedData()
         string originalEnv = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
         Environment.SetEnvironmentVariable("ASPNETCORE_ENVIRONMENT", null);
 
+        Environment.SetEnvironmentVariable("ASPNETCORE_URLS", null);
```

Comment:
Is this needed due to other tests leaking?

Reply:
Yes - buts not just the ones in the ManagementPortTest class, its all the actuators - too much change to handle by implementing IDisposable etc everywhere - cleaner to just do this in the tests that are sensitive.